backend/app/services/users/update_user_info.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Users
+  class UpdateUserInfo < Core::Service
```

Comment:
Why do you decide to rename this to UpdateUserInfo?
Will we have more separate updaters later?

Reply:
I think we add password, email, billing and other update in future with different logic

Reply:
Convinced for now 👍 
Will rethink later if needed